app/src/main/java/com/kickstarter/viewmodels/projectpage/ProjectPageViewModel.kt

```diff
@@ -1057,9 +1057,13 @@ interface ProjectPageViewModel {
         private fun managePledgeMenu(projectAndFragmentStackCount: Pair<Project, Int>): Int {
             val project = projectAndFragmentStackCount.first
             val count = projectAndFragmentStackCount.second
+            // Check the feature flag
+            // TODO: Add another validation when API is ready to add the value that PLOT is selected on the project
+            val isPledgeOverTimeEnabled = featureFlagClient.getBoolean(FlagKey.ANDROID_PLEDGE_OVER_TIME)
```

Comment:
@jlplks you need to check as well on project, the new field to be added on graphQL api `project.isPledgeOverTimeAllowed` = true.  Add in the todo which Jira ticket will be worked in at least if you wanna tackle this piece of the epic now.

Reply:
Adding it to Project data model